target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -1,7 +1,9 @@
-F:\moje_studia\Semestr VI\java\zzpj\src\main\java\com\zzpj\dtos\BookDto.java
```

Comment:
trzeba będzie dodać ten plik to .gitignore. tak samo target/zzpj-0.0.1-SNAPSHOT.jar.original